packages/ui/package.json

```diff
@@ -1,24 +1,25 @@
 {
-  "name": "ui",
+  "name": "@supabase/ui",
```

Comment:
I think we avoided doing this because there is actually a package on npm called @supabase/ui